### ISSUE TITLE
Install fails on OSX

### ISSUE THREAD

**Issue by PiedPieper** [pubkey/eth-crypto#84]
## Case
bug
## Issue
`npm install --save eth-crypto` requires node-gyp rebuild, so it will fail on node v 12 on macos https://github.com/barrysteyn/node-scrypt/issues/192

**Comment by pubkey** [comment]
@PiedPieper  With what does the install fail? Can you post the output please.

**Comment by m-yahya** [comment]
I'm trying to experiment with `eth-crypto` on windows.
It is successfully installed.
But, I'm getting following error while executing some operations:
```
{ Error: Cannot find module './build/Release/ecdh'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (C:\Users\muhammad.yahya\Desktop\data-encryption\node_modules\eccrypto\index.js:18:14)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3) code: 'MODULE_NOT_FOUND' }
Reverting to browser version
```
**Dependencies:**
node: `10.16.3`
eth-crypto: `^1.4.0`

**Sample code:**
```
const EthCrypto = require('eth-crypto');

const alice = EthCrypto.createIdentity();
const bob = EthCrypto.createIdentity();
const secretMessage = 'My name is Satoshi Buterin';

const test = async function () {
    const signature = EthCrypto.sign(
        alice.privateKey,
        EthCrypto.hash.keccak256(secretMessage)
    );
    const payload = {
        message: secretMessage,
        signature
    };
    const encrypted = await EthCrypto.encryptWithPublicKey(
        bob.publicKey, // by encryping with bobs publicKey, only bob can decrypt the payload with his privateKey
        JSON.stringify(payload) // we have to stringify the payload before we can encrypt it
    );

    console.log(encrypted);
}
```
**Steps to reproduce:**
1. init npm project
2. install the dependency
3. paste the above code in `index.js`
4. run `node index.js`

**Comment by pubkey** [comment]
I'm closing this since it's a duplicate of #47

**Comment by m-yahya** [comment]
I could not find the clear direction. can you please say a few words how to fix this?

**Comment by pubkey** [comment]
This cannot be fixed in this module. Wait for https://github.com/bitchan/eccrypto/issues/42
